models/Device.js

```diff
@@ -168,7 +168,7 @@ module.exports = function(sequelize, DataTypes) {
   };
 
   Device.userPermissions = async function(user) {
-    if (user.superuser) {
+    if (user.isAdmin()) {
```

Comment:
It might be better to call this `hasGlobalWrite` to be consistent with `hasGlobalRead`. This makes the meaning clearer too ("admin" is a bit vague).